assets/src/edit-story/components/form/numeric.js

```diff
@@ -30,7 +30,10 @@ import { __ } from '@wordpress/i18n';
 /**
  * Internal dependencies
  */
-import { Input } from '../form';
+import { Input, MULTIPLE_VALUE } from '../form';
+import useFormContext from './useFormContext';
+
+const DECIMAL_POINT = (1.1).toLocaleString().substring(1, 2);
```

Comment:
Interesting, would have thought there'd be a more standardized way how to get this :) Or perhaps that is the way. TIL

Reply:
[`Intl.Numberformat.prototype.formatToParts()`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/NumberFormat/formatToParts) would be the standardized way. But this probably works just as fine.

Reply:
I think that's not supported on Safari12.